src/System.Drawing.Design.Primitives/src/System/Drawing/Design/ToolboxItemCollection.cs

```diff
@@ -0,0 +1,69 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+
+namespace System.Drawing.Design
+{
+    /// <summary>
+    /// A collection that stores <see cref='System.Drawing.Design.ToolboxItem'/> objects.
+    /// </summary>
+    [System.Security.Permissions.PermissionSetAttribute(System.Security.Permissions.SecurityAction.LinkDemand, Name = "FullTrust")]
```

Comment:
Do we need these permission attributes?

Reply:
No, we should remove all CAS related ones.